src/xenia/gpu/graphics_system.cc

```diff
@@ -122,7 +124,8 @@ X_STATUS GraphicsSystem::Setup(cpu::Processor* processor,
   vsync_worker_running_ = true;
   vsync_worker_thread_ = kernel::object_ref<kernel::XHostThread>(
       new kernel::XHostThread(kernel_state_, 128 * 1024, 0, [this]() {
-        uint64_t vsync_duration = cvars::vsync ? 16 : 1;
+        uint64_t vsync_duration =
+            cvars::vsync ? std::max<uint64_t>(5, cvars::vsync_interval) : 1;
```

Comment:
Why is an arbitrary limit needed (that isn't just prevention of division by zero)? 1 is a valid value, 2…4 should likely be as well. Possibly 0 even considering there's an explicit 1ms sleep.

Reply:
@JoelLinn

Reply:
I remember we discussed this but have to look it up again 🙃